code/modules/wiremod/shell/scanner.dm

```diff
@@ -0,0 +1,56 @@
+/**
+ * # Scanner
+ *
+ * A handheld device which scans things.
+ */
+/obj/item/scanner
+	name = "scanner"
+	icon = 'icons/obj/assemblies/electronic_setups.dmi'
+	icon_state = "setup_small_hook"
+	item_state = "electronic"
+	//worn_icon_state = "electronic"		//remember to change it later lol
```

Comment:
:flushed:

Reply:
the compact remote and controller shells have this too so I just left it

Reply:
delete it, we're not using this

Reply:
sure